src/tito/tagger/cargotagger.py

```diff
@@ -0,0 +1,60 @@
+import re
+import os
+
+from tito.common import debug
+
+
+class CargoTagger:
+    """
+    Cargo is package manager for the Rust programming
+    language: http://doc.crates.io/manifest.html
+    It uses Cargo.toml file as its configuration file.
+    XXX: I'm not including a Toml parser, because I
+    don't want to introduce new dependencies.
```

Comment:
Would python's INI parser read this sensibly? Looking at your tests it almost looks like it would.